src/utils/time/index.js

```diff
@@ -20,24 +20,24 @@ export const calculateTime = (time, formatForChatTab = false) => {
   }
 
   if (minutes < 60) {
-    return minutes === 0 ? '1m ago' : `${minutes}m ago`;
+    return minutes === 0 ? '1m' : `${minutes}m`;
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from returning time with the "ago" suffix to without it might cause confusion for users as they may not understand if the displayed time is relative or absolute. If this change is intentional and you want to display absolute time, consider adding a label or tooltip to clarify this to the user.

```diff
-    return minutes === 0 ? '1m' : `${minutes}m`;
+    return minutes === 0 ? '1m ago' : `${minutes}m ago`;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->